subprojects/entitled-mirror/deploy.sh

```diff
@@ -11,11 +11,14 @@ if ! oc get nodes -lentitled=true -oname; then
     exit 1
 fi
 
+oc delete --ignore-not-found=true cm ${NAME}-sync-script ${NAME}-nginx
+
+oc create -n ${NAMESPACE} cm ${NAME}-sync-script --from-file=./containers/sync/sync_commands.sh
```

Comment:
Not sure I like creating objects out of template - it makes it harder to clean with `<generate template objects using template> | oc delete -f -`

Reply:
can the template receive a file content?